include/openspace/rendering/renderengine.h

```diff
@@ -151,6 +151,8 @@ class RenderEngine : public properties::PropertyOwner {
      */
     static scripting::LuaLibrary luaLibrary();
 
+    void setDebugTextureRendering(GLuint texture);
```

Comment:
This was moved here from the `RenderableModel`? Why? Does other `Renderable`s have debug textures too?